### ISSUE TITLE
Ignore mode of introduction (M/I)

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/alien-plants-belgium#17]
Does not seem to contain more useful information (Deliberate / Accidental) than V/I, so we'll ignore this in the mapping.

@qgroom is that correct?

**Comment by qgroom** [comment]
yes, that's fine

**Comment by peterdesmet** [comment]
Please discuss with Filip if it's actually worth keeping `M/I`. Doesn't`V/I` (Food refuse, Hort.) already imply accidental vs deliberate?

**Comment by peterdesmet** [comment]
We will ignore this column and rely on `V/I`

**Comment by timadriaens** [comment]
we need to cross check first whether there are any non-release/escape pathways listed as accidental and vice versa

**Comment by LienReyserhove** [comment]
I quickly inspected the mapped data, and I think that we should include the information included in `m_i` after all. An example of our mapping of `v_i` into pathway information, with extra information of `m_i`:

m_i | v_i | cbd_standard
--- | --- | ---
D | Birdseed | contaminant_seed
A/D | Hort., timber? | contaminant_timber

The CBD category `contaminant` is associated with unintentional release (see [Saul et  al.](https://doi.org/10.1111/1365-2664.12819)). Based on this information, we should think that the mode of introduction was accidental. However, the raw dataset specifies that it could also be an deliberate introduction. We are thus missing some information, although these cases are rather rare.
@timadriaens ?

**Comment by timadriaens** [comment]
This is the agreed framework of [Harrower et al.](https://www.researchgate.net/publication/323772482_Guidance_for_interpretation_of_CBD_categories_on_introduction_pathways) on deliberate versus accidental introductions linked to the CBD pathway classification, which we also use for the TrIAS pathway indicators:

<img width="290" alt="2020-04-20 12_04_56-TSSR-2016-010CBDcategoriesonpathwaysFinal pdf" src="https://user-images.githubusercontent.com/16116743/79740428-f19b9c80-82ff-11ea-8c55-11c677d95015.png">

Of course, this is very crude and so MAP has more detailed information. `M/I` also contains a value "H": this refers to hybrids and does not say anything about intentionality actually unless the hybrid is of cultivated origin. So it is a bit of a mess actually with different kinds of information in there.

I suggest we keep this in the checklist but not for the unified (there, we base entirely on `v_i`) @LienReyserhove ?

**Comment by qgroom** [comment]
My preference on this would be to get Filip to adopt the CBD pathway categories, but I don't know if he would be up for it.

**Comment by timadriaens** [comment]
Indeed, or he can keep working like he's used to but include a mapped field himself. Also, it would be good if he could keep the regional information on introduction dates etc. We might have a jobber this summer who could do that for the current MAP version so that we can provide him with that.
